components/layers/layers.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Collapse,
+  Title,
+  Group,
+  ActionIcon,
+  Divider,
+  Loader,
+  Stack,
+  Text,
+} from '@mantine/core';
+import { Layer, LayerGroup } from '@feltmaps/js-sdk';
+import useSWR from 'swr';
+import { IconEye, IconEyeOff } from '@tabler/icons-react';
+import { assembleLayerTree, useFelt, useLiveLayer } from '../../utils/felt';
+
+const Layers: React.FC = () => {
+  const felt = useFelt();
+
+  const fetchLayersAndGroups = async () => {
+    const [layers, layerGroups] = await Promise.all([
+      felt.getLayers().then(layers => layers.filter(Boolean) as Layer[]),
```

Comment:
never seen this filter(Boolean) typescript thing before. cool!

Reply:
yeah me neither! Felt continues to deliver 😎